packages/commonwealth/client/scripts/views/components/component_kit/CWRelatedCommunityCard/CWRelatedCommunityCard.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import type ChainInfo from 'client/scripts/models/ChainInfo';
+import "./CWRelatedCommunityCard.scss"
+import { CWText } from '../cw_text';
+import { CWCommunityAvatar } from '../cw_community_avatar';
+import { ComponentType } from '../types';
+import { CWIcon } from '../cw_icons/cw_icon';
+import { pluralizeWithoutNumberPrefix } from '../../../../helpers';
+import { CWButton } from '../cw_button';
+
+type CWRelatedCommunityCardProps = {
+  chain: ChainInfo,
+  memberCount: number,
+  threadCount: number,
+}
+
+export const CWRelatedCommunityCard = ( props: CWRelatedCommunityCardProps) => {
+  const { chain, memberCount, threadCount } = props;
+
+  const prettifyDescription = (community: ChainInfo) => {
+    return (
+      community.description[community.description.length - 1] === '.'
+      ? community.description
+      : `${community.description}.`
+    )
+  }
```

Comment:
this is a util function and can be easily outside of the component, ideally in some `utils` file

Reply:
Good suggestion and done!